lib/datasource/local/settings_storage.dart

```diff
@@ -182,9 +202,18 @@ class _SettingsStorage {
 
   void savePasscode(String? passcode) => this.passcode = passcode;
 
-  void saveAccount(String accountName, String privateKey) {
+  Future<void> saveAccount(String accountName, String privateKey) async {
     _accountName = accountName;
-    this.privateKey = privateKey;
+    _privateKey = privateKey;
```

Comment:
ok so _privateKey is the current private key? That sort of makes sense we can keep it for now, but it will probably get removed later on.